### ISSUE TITLE
Have an image that displays the characters for the various modes as an image

### ISSUE THREAD

**Issue by Yazwh0** [X16Community/x16-docs#113]
Including the hex value which is to be written to VRAM.

Eg:

![image](https://github.com/X16Community/x16-docs/assets/90315107/05e5877e-fb71-4560-b876-92dd942ea857)

**Comment by JimmyDansbo** [comment]
https://cx16.dk/veratext/ 

Feel free to use

**Comment by JimmyDansbo** [comment]
Maybe the appendix I added in https://github.com/X16Community/x16-docs/pull/249 can close this?

**Comment by tomxp411** [comment]
Yes, it seems like this does what OP wants, and it's a useful addition. When #249 is closed, this issue should be closed with it.

**Comment by tomxp411** [comment]
Done in PR #249
